tests/TTIR/test_conv2d.py

```diff
@@ -10,35 +10,35 @@
 
 
 @pytest.mark.parametrize(
-    "batch_size, output_channels, input_channels, input_height, input_width, filter_height, filter_width, stride_h, stride_w, padding",
+    "batch_size, output_channels, input_channels, input_height, input_width, filter_height, filter_width, stride_h, stride_w, padding, pcc",
     (
         # RESNET
-        (1, 64, 3, 224, 224, 7, 7, 2, 2, 3),
-        (1, 256, 64, 56, 56, 1, 1, 1, 1, 0),
-        (1, 64, 64, 56, 56, 1, 1, 1, 1, 0),
-        (1, 64, 64, 56, 56, 3, 3, 1, 1, 1),
-        (1, 64, 256, 56, 56, 1, 1, 1, 1, 0),
-        (1, 512, 256, 56, 56, 1, 1, 2, 2, 0),
-        (1, 128, 256, 56, 56, 1, 1, 2, 2, 0),
-        (1, 128, 128, 28, 28, 3, 3, 1, 1, 1),
-        (1, 512, 128, 28, 28, 1, 1, 1, 1, 0),
-        (1, 128, 512, 28, 28, 1, 1, 1, 1, 0),
-        # (1, 1024, 512, 28, 28, 1, 1, 2, 2, 0), Requires block sharding
-        (1, 256, 512, 28, 28, 1, 1, 2, 2, 0),
-        (1, 256, 256, 14, 14, 3, 3, 1, 1, 1),
-        (1, 1024, 256, 14, 14, 1, 1, 1, 1, 0),
-        (1, 256, 1024, 14, 14, 1, 1, 1, 1, 0),
-        # (1, 2048, 1024, 14, 14, 1, 1, 2, 2, 0), Requires block sharding
-        # (1, 512, 1024, 14, 14, 1, 1, 2, 2, 0), Requires block sharding
-        # (1, 512, 512, 7, 7, 3, 3, 1, 1, 1), Requires block sharding
-        # (1, 2048, 512, 7, 7, 1, 1, 1, 1, 0), Requires block sharding
-        # (1, 512, 2048, 7, 7, 1, 1, 1, 1, 0), Requires block sharding
+        (1, 64, 3, 224, 224, 7, 7, 2, 2, 3, 0.95),
+        (1, 256, 64, 56, 56, 1, 1, 1, 1, 0, 0.95),
+        (1, 64, 64, 56, 56, 1, 1, 1, 1, 0, 0.95),
+        (1, 64, 64, 56, 56, 3, 3, 1, 1, 1, 0.95),
+        (1, 64, 256, 56, 56, 1, 1, 1, 1, 0, 0.95),
+        (1, 512, 256, 56, 56, 1, 1, 2, 2, 0, 0.95),
+        (1, 128, 256, 56, 56, 1, 1, 2, 2, 0, 0.95),
+        (1, 128, 128, 28, 28, 3, 3, 1, 1, 1, 0.95),
+        (1, 512, 128, 28, 28, 1, 1, 1, 1, 0, 0.95),
+        (1, 128, 512, 28, 28, 1, 1, 1, 1, 0, 0.95),
+        (1, 1024, 512, 28, 28, 1, 1, 2, 2, 0, 0.95),
+        (1, 256, 512, 28, 28, 1, 1, 2, 2, 0, 0.95),
+        (1, 256, 256, 14, 14, 3, 3, 1, 1, 1, 0.95),
+        (1, 1024, 256, 14, 14, 1, 1, 1, 1, 0, 0.95),
+        (1, 256, 1024, 14, 14, 1, 1, 1, 1, 0, 0.95),
+        (1, 2048, 1024, 14, 14, 1, 1, 2, 2, 0, 0.95),
+        (1, 512, 1024, 14, 14, 1, 1, 2, 2, 0, 0.95),
+        (1, 512, 512, 7, 7, 3, 3, 1, 1, 1, 0.95),
+        (1, 2048, 512, 7, 7, 1, 1, 1, 1, 0, 0.95),
+        (1, 512, 2048, 7, 7, 1, 1, 1, 1, 0, 0.8),  # Lower PCC than the rest.
```

Comment:
Do you maybe know why this case requires such a low PCC? I'd rather mark this one as skipped until ttnn is improved, than having to parametrize pcc like this.

Reply:
No idea. Just reran it on another machine, same thing.

Will skip it instead of parametrizing PCC.